makefile.cargo

```diff
@@ -23,6 +23,12 @@ AR ?= ar
 
 endif
 
+ifeq ("$(MSYSTEM)","")
+CXXFLAGS += -fPIC
```

Comment:
I'm pretty sure -fPIC is a no-op on Windows, due to the ABI.  This builds, at least.